cms/elm/Main.elm

```diff
@@ -0,0 +1,15 @@
+module Main exposing (main)
+
+import Types exposing (..)
```

Comment:
I think ```exposgin (..)``` will expose all the functions, if possible I think it's better to expose only the ones we want.